index.bs

```diff
@@ -653,6 +654,8 @@ Each {{XRSession}} has a <dfn for=XRSession>set of granted features</dfn>, which
 
 The <dfn attribute for="XRSession">enabledFeatures</dfn> attribute returns the features in the [=XRSession/set of granted features=] as a new array of {{DOMString}}s.
 
+The <dfn attribute for="XRSession">isSystemKeyboardSupported</dfn> attribute indicates that the {{XRSystem}} has the ability to display the system keyboard while the {{XRSession}} is active.
```

Comment:
probably should list the various conditions under which this may happen

Reply:
Do you know where this is called out in the HTML spec? https://html.spec.whatwg.org/multipage/interaction.html#dom-focus-dev does seem to mention showing the keyboard?

I'm also unsure if we should call out the conditions. In theory, this feature should already work.

Reply:
Yes, but "already work" _when_? Clearly we don't expect the keyboard to be popping up randomly, nor do we expect it to show up in one specific case and not elsewhere.

Reply:
I clarified the attribute a bit.